src/utils/getSelectedEpisodeOrSearchedEpisodes.ts

```diff
@@ -0,0 +1,19 @@
+import { IEpisode } from "../types";
+
+export function getSelectedEpisodeOrSearchedEpisodes(
```

Comment:
The function name is misleading as this function doesn't search/filter the episodes based on the searchTerm